config/gpt2_nano_mixture.yaml

```diff
@@ -0,0 +1,30 @@
+data:
+  configs:
+    wikitext:
+      id: dlwh/wikitext_103_detokenized
+    w2:
+      id: dlwh/wikitext_103_detokenized
```

Comment:
If we use different names, this will make two eval loss curves, despite that these are the same datasets. Nice trick.